components/nexusoperations/executors.go

```diff
@@ -309,6 +333,25 @@ func (e taskExecutor) saveResult(ctx context.Context, env hsm.Environment, ref h
 				return hsm.TransitionOutput{}, err
 			}
 			if result.Pending != nil {
+				var links []*commonpb.Link
+				for _, nexusLink := range result.Links {
+					switch nexusLink.Type {
+					case string((&commonpb.Link_WorkflowEvent{}).ProtoReflect().Descriptor().FullName()):
+						link, err := ConvertNexusLinkToLinkWorkflowEvent(nexusLink)
+						if err != nil {
+							// silently ignore for now
```

Comment:
Would have been better to log so there's indication but let's do that in a follow up PR.